embed-code-go/embedding/embedding_processor.go

```diff
@@ -119,10 +120,47 @@ func EmbedAll(config configuration.Configuration) {
 	documentationRoot := config.DocumentationRoot
 	docPatterns := config.DocIncludes
 	for _, pattern := range docPatterns {
-		documentationFiles, _ := filepath.Glob(filepath.Join(documentationRoot, pattern))
+		globString := strings.Join([]string{documentationRoot, pattern}, "/")
+		documentationFiles, _ := doublestar.FilepathGlob(globString)
 		for _, documentationFile := range documentationFiles {
 			processor := NewEmbeddingProcessor(documentationFile, config)
 			processor.Embed()
 		}
 	}
 }
+
+// Raises an error if the documentation files are not up-to-date with code files.
```

Comment:
We don't explain this "up-to-date" anywhere. Can we do that somewhere on a higher level? 

Right now, I could not tell for sure what it is even if I wanted to.